mongomock/collection.py

```diff
@@ -1788,9 +1788,15 @@ def with_options(
     def rename(self, new_name, **kwargs):
         self.database.rename_collection(self.name, new_name, **kwargs)
 
-    def bulk_write(self, operations):
+    def bulk_write(self, requests, ordered=True, bypass_document_validation=False, session=None):
+        if not ordered:
+            raise NotImplementedError('Working in unordered mode is unsupported')
```

Comment:
Please make sure that `mongomock` is the obvious culprint: "Unordered mode is a valid MongoDB operation, however it is not supported by mongomock yet". Same below.

Reply:
I added a note about sessions only being valid for MongoDB 3.6+ for maximum clarity as well.